csr/README.md

```diff
@@ -0,0 +1,58 @@
+vrnetlab / Cisco CSR1000v
+===========================
+This is the vrnetlab docker image for Cisco CSR1000v.
+
+To make sure that we get the console output on serial, we are first building 
+a docker image to boot up the CSR with a mounted iso image including the
+bootstrap configuration.
+After the installtion process we are using docker commit to save the state.
+
+Building the docker image
+-------------------------
+
+Put the .qcow2 file in this directory and run `make docker-image` and
+you should be good to go. The resulting image is called `vr-csr`. You can tag
+it with something else if you want, like `my-repo.example.com/vr-csr` and then
+push it to your repo. The tag is the same as the version of the CSR image, so
+if you have csr1000v-universalk9.16.04.01.qcow2 your final docker image will be called
+vr-csr:16.04.01
+
+It's been tested to boot and respond to SSH with:
+
+ * 16.03.01a (csr1000v-universalk9.16.03.01a.qcow2)
+ * 16.04.01 (csr1000v-universalk9.16.04.01.qcow2)
+
+Usage
+-----
+```
+docker run -d --privileged --name my-csr-router vr-csr
+```
+
+Interface mappning
+
+| vr-csr | vr-xcon |
+| :---:  |  :---:  |
+| Gi2 | 10 |
+| Gi3 | 1 |
+| Gi4 | 2 |
+| Gi5 | 3 |
+| Gi6 | 4 |
+| Gi7 | 5 |
+| Gi8 | 6 |
+|Gi9 | 7 |
+|Gi10| 8 |
+|Gi11| 9 |
+
+System requirements
+-------------------
+CPU: 1 core
+
+RAM: 4GB
```

Comment:
Does it really require this much? It's using 4 now because that's the default of the VM class.